File: modul6/Python/PRAK605-2310817210029-Putra_Whyra_Pratama_Setiawan.py
import numpy as np
n = int(input())
matriksA = [[0]*n]*n
matriksB = [[0]*n]*n
print("Matriks A")
for i in range(0, n):
    matriksA[i] += np.array(list(map(int, input().split( ))))
print("Matriks B")
for i in range(0, n):
    matriksB[i] += np.array(list(map(int, input().split( ))))
print("Matriks AXB")
matriksC = np.dot(matriksA, matriksB)
for hasil in matriksC:
    print(*hasil)

File: modul6/C/PRAK605-2310817210029-Putra_Whyra_Pratama_Setiawan.c
#include <stdio.h>
int main() {
    int n;
    scanf("%d", &n);
    int matriksA[n][n], matriksB[n][n], matriksC[n][n];
    printf("Matriks A\n");
    for(int i = 0;i < n;i++) {
        for(int j = 0;j < n;j++) {
            scanf("%d", &matriksA[i][j]);
        }
    }
    printf("Matriks B\n");
    for(int i = 0;i < n;i++) {
        for(int j = 0;j < n;j++) {
            scanf("%d", &matriksB[i][j]);
        }
    }
    printf("Matriks AXB\n");
    for(int i = 0;i < n;i++) {
        for(int j = 0;j < n;j++) {
            matriksC[i][j] = 0;
            for(int k = 0;k < n;k++) {
                matriksC[i][j] += matriksA[i][k] * matriksB[k][j];
            }
            printf("%d ", matriksC[i][j]);
        }
        printf("\n");
    }
}